### DISCUSSION TITLE
1.38.0

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#22869] [General]
Feel free to discuss the [v1.38.0](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.38.0) release here!

**Comment by altopais** [comment]
Moes 2 gang smart dimmer stopped dimming on one gang. L1 is not dimming, L2 is dimming. 
It's reported as TS0601_dimmer_2.
In z2m logs i get only info when changing brightness on L1, no errors or warnings.

**Comment by dz1984sg** [comment]
Since the update, my device list is empty, but all devices are still connected and working! 
![Screenshot 2024-06-02 090133](https://github.com/Koenkk/zigbee2mqtt/assets/157277604/802ba9dc-bbf0-4a50-a8a8-1a7f506c460d)

**Reply by PeterKawa** [reply]
I've still got all the values after updating. Did you force refresh the page, restart or fully rebooted HA or Z2M?

**Reply by dz1984sg** [reply]
already tried everything ,,,, as already written, all devices already paired work

**Reply by dz1984sg** [reply]
error was in the configuration.yaml .... whatever was wrong with it .... deleted all the devices there and now I see everything again .... just reassign the names and that's it

**Reply by LaurentChardin** [reply]
It usually happens when you switch to ember the first time, and it actually re-creates its state from the backup : for some minutes, the information is empty. But once the devices are backup, the page will be updated with the proper informations.

**Reply by dz1984sg** [reply]
there was no change .... that's why i share the information about the error .... because it makes no sense and there was no reason .....

no new gears were learned. after the update everything ran for 2 hours ca..... only then was the list empty! ..

**Reply by LaurentChardin** [reply]
All of those information are read from the MQTT bridge/info topic which are sent to react through websocket. 
So for some reasons, you didnt get the message. Restarting z2m in the background will resent the bridge/info message to the frontend (as this is publish at startup, and when device changes).
You should still see the message in MQTT  anyway as this is a retained message : if this is not even there, then yes, there is something wrong. If it is there ,but you dont see it in the frontend : probably something happened during the sync/start of react app.

**Comment by Dydas** [comment]
After the update Zigbee2MQTT cannot start. Errors in the logs: "Error relocating /bin/bash"

![image](https://github.com/Koenkk/zigbee2mqtt/assets/34096447/c3207fa7-4eb0-4d39-a89f-df8191f6a2a1)

**Comment by mmirannalti** [comment]
I have a few Bosch BMCT-SLZ.
As from notes, I have reconfigured them.
It worked in z2m, but in HA the device don't reflect the attributes exposed in z2m.
Is there a way to force the publish of the device structure to HA?

Thanks

**Comment by Fullmojjo** [comment]
Müller Licht Tint remote 404049D is stopped since update sonoff ember driver. Any update ?
